### PR TITLE
Feat/open dosm dashboards

### PR DESCRIPTION
**Pull request by amalasyrafs**
- added iip dashboard page in pages/dashboard
- added iip dashboard component in dashboard/national-accounts
- added national-accounts key in datagovmy/ui/types
- added i18n for iip dashboard

### CODE REVIEW

**Comment by amalasyrafs** [comment]
@irfancoder , not sure if the recession part of the dashboard was done correctly or not... hope you could verify this.

**Comment by amalasyrafs** [comment]
@irfancoder , also from the figma, the agency is National Accounts Division, now just using DoSM. do i have to add the agency ?

File: apps/app/dashboards/national-accounts/iip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    "other",
+    "porfolio_equity",
+    "portfolio_debt",
+    "portfolio_total",
+    "reserve",
+  ]);
+
+  useWatch(() => {
+    setData("minmax", [0, timeseries.data[data.index_type.value].x.length - 1]);
+  }, [data.index_type]);
```
Comment (irfancoder) [inline apps/app/dashboards/national-accounts/iip/index.tsx:140]: I think can remove this. Iinm, the reset is handled by the Slider component itself.

**Comment by irfancoder** [comment]
> @irfancoder , not sure if the recession part of the dashboard was done correctly or not... hope you could verify this.

Looks off. There should be a recession shading in the chart.


> @irfancoder , also from the figma, the agency is National Accounts Division, now just using DoSM. do i have to add the agency ?

Keep to DOSM. National Accounts Division is under DOSM. 


Some things to note:

1. Add the i18n namespace inside `next-i18next.config.js`

**Comment by amalasyrafs** [comment]
Amended changes from feedbacks:

- [x] remove useWatch in iip dashboard - Irfan
- [x] add recession shader in the charts 
- [x] Add the i18n namespace inside next-i18next.config.js - Irfan
- [x] moved iip from datagovmy to opendosm
- [x] change chart axis to 1dp - Thevesh
- [x] rearrange charts to correct order according to Figma - Thevesh

**Comment by irfancoder** [comment]
One last change;

- Update the `dashboard/index` page. Add the new dashboard in

**Comment by amalasyrafs** [comment]
> One last change;
> 
> * Update the `dashboard/index` page. Add the new dashboard in

Done!

**Review by irfancoder — APPROVED** [review]

**Pull request merged**